src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const spyder = document.querySelector('.spider');
+
+spyder.style.transform = 'translate(-50%, -50%)';
+spyder.style.top = '50%';
+spyder.style.left = '50%';
```

Comment:
smart)
Yes, this task can be easily done with pure CSS, but not now. Lets do it with JS. (use wall and spider size)